Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -207,254 +207,248 @@ export default class ReportsView extends Vue {
-                <b-col class="col-12 col-md-10 col-lg-9 column-wrapper">
-                    <page-title title="Export Records" />
-                    <div class="my-3 px-3 py-4 form">
+        <b-row>
+            <b-col class="col-12 col-lg-9 column-wrapper">
+                <page-title title="Export Records" />
+                <div class="my-3 px-3 py-4 form">
+                    <b-row>
+                        <b-col>
+                            <label for="reportType"> Record Type </label>
+                        </b-col>
+                    </b-row>
+                    <b-row align-h="between" class="py-2">
+                        <b-col class="mb-2" md="12" lg="">
+                            <b-form-select
+                                id="reportType"
+                                v-model="reportType"
+                                data-testid="reportType"
+                                :options="reportTypeOptions"
+                            >
+                            </b-form-select>
+                        </b-col>
+                        <b-col class="p-0 px-3" cols="auto">
+                            <hg-button
+                                v-b-toggle.advanced-panel
+                                variant="link"
+                                data-testid="advancedBtn"
+                            >
+                                Advanced
+                            </hg-button>
+                            <hg-button
+                                variant="primary"
+                                data-testid="exportRecordBtn"
+                                class="mb-1 ml-2"
+                                :disabled="
+                                    !reportType ||
+                                    isLoading ||
+                                    !patientData.hdid
+                                "
+                                @click="showConfirmationModal"
+                            >
+                                Download PDF
+                            </hg-button>
+                        </b-col>
+                    </b-row>
+                    <b-row v-if="reportFilter.hasActiveFilter()" class="pb-2">
+                        <b-col v-if="reportFilter.hasDateFilter()">
+                            <div><strong>Date Range</strong></div>
+                            <b-form-tag
+                                variant="light"
+                                class="filter-selected"
+                                title="From"
+                                data-testid="clearFilter"
+                                :pill="true"
+                                @remove="clearFilterDates"
+                            >
+                                <span data-testid="selectedDatesFilter"
+                                    >{{
+                                        reportFilter.startDate
+                                            ? `From ${formatDateLong(
```

Comment:
The `formatDateLong()` function could be removed from this file and its invocations replaced with direct calls to `DateWrapper.format()`, like you did in other files.

Reply:
good call, It's gone.

Reply:
Double-check - this is actually not possible since the vue template does not have access to the included libraries in the file, instead, it only has access to the methods defined in the component.
TL;DR Reverted, vue cant do that